### PR TITLE
docs: translate to zh-CN

### PR DESCRIPTION
**Pull request by edison1105**
translate docs to Chinese.
Welcome to calibration and review.

### CODE REVIEW

**Comment by posva** [comment]
Oh, nice! There is a lot of traffic from China in the docs, so I think this would be nice to be integrated in the repository but it will have to be reviewed by some of the core team so it might take some time

**Comment by edison1105** [comment]
> Oh, nice! There is a lot of traffic from China in the docs, so I think this would be nice to be integrated in the repository but it will have to be reviewed by some of the core team so it might take some time


Got it.

**Comment by edison1105** [comment]
Hi @posva
This PR is readly for review.

**Comment by posva** [comment]
Thanks a lot! I will relay the message to get some reviews!

File: docs/zh/guide/advanced/composition-api.md
Comment (antfu) [inline docs/zh/guide/advanced/composition-api.md:87]:
```suggestion
Vue Router 将 RouterLink 的内部行为作为一个组合式 API 函数公开。它提供了与 [`v-slot` API](/zh/api/#router-link-s-v-slot) 相同的访问属性：
```

File: docs/zh/guide/advanced/data-fetching.md
Comment (antfu) [inline docs/zh/guide/advanced/data-fetching.md:74]:
```suggestion
通过这种方式，我们在导航转入新的路由前获取数据。我们可以在接下来的组件的 `beforeRouteEnter` 守卫中获取数据，当数据获取成功后只调用 `next` 方法：
```

File: docs/zh/guide/advanced/dynamic-routing.md
Comment (antfu) [inline docs/zh/guide/advanced/dynamic-routing.md:32]:
```suggestion
记住，如果你需要等待新的路由显示，可以使用 `await router.replace()`。
```

File: docs/zh/guide/advanced/dynamic-routing.md
Comment (antfu) [inline docs/zh/guide/advanced/dynamic-routing.md:37]:
```suggestion
如果你决定在导航守卫内部添加或删除路由，你不应该调用 `router.replace()`，而是通过返回新的位置来触发重定向：
```

File: docs/zh/guide/advanced/navigation-failures.md
Comment (antfu) [inline docs/zh/guide/advanced/navigation-failures.md:8]:
```suggestion
- 一个[导航守卫](./navigation-guards.md)通过调用 `return false` 中断了这次导航
- 当前的导航守卫还没有完成时，一个新的导航守卫会出现了
- 一个[导航守卫](./navigation-guards.md)通过返回一个新的位置，重定向到其他地方 (例如， `return '/login'`)
```

File: docs/zh/guide/advanced/navigation-guards.md
Comment (antfu) [inline docs/zh/guide/advanced/navigation-guards.md:3]:
```suggestion
正如其名，vue-router 提供的导航守卫主要用来通过跳转或取消的方式守卫导航。这里有很多方式植入路由导航中：全局的，单个路由独享的，或者组件级的。
```

File: docs/zh/guide/advanced/scroll-behavior.md
Comment (antfu) [inline docs/zh/guide/advanced/scroll-behavior.md:109]: 😉

**Review by antfu — APPROVED** [review]
@posva Did a quick and rough overview on GitHub, I think the translations are quite natural and there isn't much I can complain about. Just a few nitpicking on formatting, but I think it's a pretty good starting point for the Chinese doc and ready to merge as long as the build process is alright.

@edison1105 Well done! I have left a few comments about the formatting glitches, please have a look. I think it's a bit redundant to have spaces around `。`，` since they are already full-width characters. I address a few but you might like to have a global search over that.

**Comment by edison1105** [comment]
@antfu thanks for your review.I will check the things you mentioned above in each file.

File: docs/zh/guide/advanced/dynamic-routing.md
Comment (meteorlxy) [inline docs/zh/guide/advanced/dynamic-routing.md:51]:
```suggestion
因为我们是在重定向，所以我们是在替换正在进行的导航，实际上行为就像之前的例子一样。而在实际场景中，添加路由的行为更有可能发生在导航守卫之外，例如，当一个视图组件挂载时，它会注册新的路由。
```

Comment (meteorlxy) [reply docs/zh/guide/advanced/dynamic-routing.md:51]: > 因为我们是在重定向，所以我们是在替换正在进行的导航，实际上行为就像之前的例子一样

这句感觉可以改改

File: docs/zh/introduction.md
Comment (meteorlxy) [inline docs/zh/introduction.md:3]:
```suggestion
Vue Router 是 [Vue.js](http://v3.vuejs.org)的官方路由。它与 Vue.js 核心深度集成，让用 Vue.js 构建单页应用变得轻而易举。功能包括：
```

**Review by meteorlxy — CHANGES_REQUESTED** [review]
Did a partial review

**Comment by posva** [comment]
Let's merge this to make it available for all Chinese users and they will be able to propose changes as they see.

I have seen a few usability issues when it comes to language selection but I will have to fix those in Vitepress

Thanks a lot @edison1105 @antfu @meteorlxy

**Pull request merged**
